.github/workflows/persistent-devnet.yml

```diff
@@ -96,8 +97,8 @@ jobs:
     - name: Deploy
       run: |
         # For the love of God can anyone tell me why github runners are not stateless.
-        nixops check --include composable-persistent-devnet
-        nixops deploy --allow-recreate --confirm --deployment devnet-gce --debug --show-trace --option narinfo-cache-negative-ttl 0 --include composable-persistent-devnet
+        nixops check --include composable-persistent-devnet || true
```

Comment:
i think `|| true` should probably be removed?

Reply:
No it shouldn't because the exit code of check is very opiniated :)